lib/normalize-query-options/spatial-reference.js

```diff
@@ -10,7 +10,9 @@ const schema = Joi.alternatives(
     wkid: Joi.number().integer().optional(),
     latestWkid: Joi.number().integer().optional(),
     wkt: Joi.string().optional()
-  }).or('wkid', 'latestWkid', 'wkt').required()
+  }).or('wkid', 'latestWkid', 'wkt')
+    .required()
+    .unknown()
```

Comment:
How exactly does the validation translate here? Based on the documentation here: https://joi.dev/api/?v=17.6.0#objectorpeers-options seems like the `.or()` makes it so that at least one is required then? But their documentation also makes it seem like the `required()` is not necessary?

Reply:
I think you are right, `.required()` not needed here.  Removed.